packages/usa-in-page-navigation/src/index.js

```diff
@@ -187,20 +187,29 @@ const getSectionId = (value) => {
   return id;
 };
 
+/**
+ * Recursively calculate a heading element's offset from the top of the page.
+ *
+ * @param {HTMLElement} el A heading element
+ */
+const getTotalElementOffset = (el) =>
+  el.offsetTop + (el.offsetParent && getTotalElementOffset(el.offsetParent));
+
```

Comment:
**question:** Is the el.offsetParent && a truthy check to check for parent offset? If so, we might want to make the check a bit more human-readable to match existing USWDS patterns.

**suggestion:** Opt for `if` statement:

```suggestion
const getTotalElementOffset = (el) => {
  if (el.offsetParent) {
    return el.offsetTop +  getTotalElementOffset(el.offsetParent);
  } 

  return el.offsetTop
}
```

Curious if the rest of the team has another preference!